src/components/DocumentationTopic/PrimaryContent/DeclarationSourceLink.vue

```diff
@@ -0,0 +1,60 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <a
```

Comment:
Is it worth to use the Reference component instead? Will always be external links?

Reply:
Yes, it will always point to Github :) there is no need for a custom component I think.